AwsEncryptionSDK/runtimes/net/CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## 4.1.0
+
+### Fixes
+
+* fix(CI): Daily CI uses correct workflow [(#641)](https://github.com/aws/aws-encryption-sdk-dafny/commit/771835e22f6ef3c3b34d0891fb61cb1a49bcf855)
+* fix: throw an exception when MemoryStream instance has an empty backing array [(#633)](https://github.com/aws/aws-encryption-sdk-dafny/commit/550c714743e84f93d09900b3338f59d0a54bb3ce)
+* chore(ci): fix role to assume [(#622)](https://github.com/aws/aws-encryption-sdk-dafny/commit/c1f04fc41093593748f16da80d893c2ec5325545)
+
+### Features
+
+* feat: enforce input constraints [(#646)](https://github.com/aws/aws-encryption-sdk-dafny/commit/10daadfa19db0e43fc0cc6d7b989f1fb477a22b0)
```

Comment:
Could we explain this a bit more? Like in https://github.com/aws/aws-database-encryption-sdk-dynamodb/commit/5954fb33fdc8091191eae9293e3de5712c4e1ed2